docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+
+
+### Approaches
+
+There are at least three approaches for external WebLogic EJB or JMS client access to a Kubernetes hosted WebLogic cluster. These include [Federation](#federation), [Load Balancer Tunneling](#load-balancer-tunneling), and [Kubernetes Node Ports](#kubernetes-node-ports).
+
+> __NOTE:__ This FAQ is for remote EJB and JMS clients - not JTA clients. The Operator does not currently support external WebLogic JTA access to a WebLogic cluster, as (a) external JTA access requires each server in the cluster to be individually addressable by the client, but this conflicts with (b) the current Operator requirement that a network channel in a cluster have the same port across all servers in the cluster.
+
+#### Federation
+
+- Deploy the remote clients inside another Kubernetes cluster.
+
+- Federate the two clusters so that they effectively share the same DNS namespace. See [Federation](https://kubernetes.io/docs/concepts/cluster-administration/federation/) in the Kubernetes documentation.  Note that federation is a relatively new Kubernetes feature as of 2019.
+
+- If the addresses (URLs) for remote access from the client cluster to the original cluster are not the same as the addresses used by clients running within the original cluster itself, then it is often necessary to configure a custom channel for the T3 protocol in the original cluster.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+
+#### Load Balancer Tunneling
+
+
+- Configure a custom channel for the T3 protocol in WebLogic that (a) enables HTTP Tunneling, and (b) specifies an external address and port that correspond to the address and port remote clients will use access the load balancer.  See  [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+- Setup a load balancer that redirects HTTP traffic to the custom channel. For a discussion of load balancers see [Ingress]({{<relref "/userguide/managing-domains/ingress/_index.md">}}).
+
+- Remote clients can then access the custom channel using an 'http://' URL instead of a 't3://' URL.
+
+- Review the [Security Notes](#security-notes) below.
+
+#### Kubernetes Node Ports
+
+- Configure a custom channel for the T3 protocol in WebLogic that specifies an external address and port that are suitable for remote client use.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+- Deploy a Kubernetes NodePort to publicly expose the WebLogic ports. See [Setting up a NodePort](#setting-up-a-nodeport).
+
+- Review the [Security Notes](#security-notes) below.
+
+### Adding a WebLogic Custom Channel
+
+#### When is a WebLogic Custom Channel needed?
+
+WebLogic implicitly creates a multi-protocol default network channel that spans the Listen Address and Port fields specified on each server in the cluster. Here are some reasons you may need to configure an additional dedicated WebLogic custom channel to handle remote EJB or JMS client network traffic:
```

Comment:
I would not say "here are some reasons" - implies there are others, just say something like "you need to do x when:"

Reply:
attempted a rewording